entity-service-impl/src/main/java/org/hypertrace/entity/data/service/EntityDataServiceImpl.java

```diff
@@ -720,18 +721,18 @@ private JSONDocument convertEntityRelationshipToDocument(
     }
   }
 
-  private void sentDeleteNotification(String tenantId, java.util.Collection<Entity> entities) {
+  private void trySendDeleteNotification(String tenantId, java.util.Collection<Entity> entities) {
```

Comment:
my last remaining comment on this, and its up to you, is that as the only consumer of `entityChangeEventGenerator`, we shouldn't have to wrap its methods. I'd push this try/catch/log into the generator itself.

Reply:
Done